### DISCUSSION TITLE
Campos adicionales

### DISCUSSION THREAD

**Discussion by CVargasGando** [Cotalker/documentation#767] [3. Ideas & Feature Requests]
Hola! Espero que tengan un lindo día :) 

Quería saber si está la opción de agregar más campos adicionales, ya que la mayoría de mis campos adicionales los ocupo para "control form" :(  
Sería relevante para que el cliente pueda elegir sus filtros en la task view. Se podrían agregar campos adicionales ocultos para los control form o alguna otra cosa que se necesite para la implementación.
<img width="1044" alt="image" src="https://user-images.githubusercontent.com/73865460/158628850-e9308761-60f6-4ced-a8c2-717c3b3846f3.png">

Muchas gracias!!

**Comment by ValeMP** [comment]
x2

**Comment by acedward** [comment]
Estos control forms son para poder mostrar y/o ocultar formularios?
